src/SeisSol.cpp

```diff
@@ -108,10 +108,12 @@ bool seissol::SeisSol::init(int argc, char* argv[])
 
 #ifdef _OPENMP
   logInfo(rank) << "Using OMP with #threads/rank:" << omp_get_max_threads();
-  logInfo(rank) << "Overall affinity            :" << parallel::Pinning::maskToString(
+  logInfo(rank) << "Overall affinity (this process)      :" << parallel::Pinning::maskToString(
       pinning.getProcessMask());
```

Comment:
Necessary?

Reply:
No, is actually really confusing when using gcc..